src/pages-and-resources/discussions/app-config-form/apps/shared/discussion-topics/DiscussionTopics.jsx

```diff
@@ -8,31 +8,36 @@ import { v4 as uuid } from 'uuid';
 
 import messages from '../messages';
 import TopicItem from './TopicItem';
-import { removeModel } from '../../../../../../generic/model-store';
+import { removeModel, updateModels } from '../../../../../../generic/model-store';
 import { updateDiscussionTopicIds } from '../../../../data/slice';
-import { updatedDiscussionTopics } from '../../../../data/thunks';
 
 const DiscussionTopics = ({ intl }) => {
   const dispatch = useDispatch();
-  const { values } = useFormikContext();
-  const [topics, setTopics] = useState(values.discussionTopics);
+  const { values: appConfig, setFieldValue } = useFormikContext();
+  const { discussionTopics, dividedDiscussionsIds } = appConfig;
+  const [topics, setTopics] = useState(discussionTopics);
 
   useEffect(() => {
-    const updatedDiscussionTopicIds = values.discussionTopics?.map(topic => topic.id);
+    const updatedDiscussionTopicIds = discussionTopics.map(topic => topic.id);
```

Comment:
Still need the "?" here?

Reply:
No because discussionTopics never be undefined. It will be empty is worse case.